### ISSUE TITLE
Data science project "Handwritten Digits Recognition"

### ISSUE THREAD

**Issue by vaishnavi-1** [Jayshah6699/datascience-mashup#18]
I want to work on this issue. The accuracy of my model was approximately **98%.** Kindly @Jayshah6699 put the labels of DWOC and level and then assign it to me. I have made a CNN model for recognising the handwritten digits and used keras, tensorflow for training the **MNIST dataset.** I had trained and tested the model on Google Colab.

**Comment by SoyabulIslamLincoln** [comment]
I worked on this project which had 98.9% accuracy on custom datasets as a public score. I made a CNN model with the help of tensorflow, keras. I worked on that project on Kaggle. It would be great if you assign me this task.

**Comment by vaishnavi-1** [comment]
@SoyabulIslamLincoln I have created this issue if you have made the model then why didn't you created the issue..due to difference of 0.9 % it is not appreciable to make you the assignee..
You should make another issue for another project.
I will also try to increase the accuracy of my model.

**Comment by SoyabulIslamLincoln** [comment]
I am really sorry @vaishnavi-1 . I thought the issue wasn't assigned or was assigned before. So i asked for that. Really sorry about that. Hope it is okay now.

**Comment by vaishnavi-1** [comment]
It's okay @SoyabulIslamLincoln.

**Comment by Jayshah6699** [comment]
@vaishnavi-1 do one thing I am assigning this issue to you and @SoyabulIslamLincoln you create a separate issue named different model for hand written digit recognition and let me first merge @vaishnavi-1 pull request and then in the folder created by @vaishnavi-1 you will add your folder that I mentioned before and then I will merge your pull request
